Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "1.5.1"
+version = "1.5.2"
```

Not super-important, but I think this counts as a feature:
```suggestion
version = "1.6.0"
```

---

:-) Thanks for merging...
At least `ChainRules.jl` is on par with Julia base now ;-)